src/main/java/net/dries007/tfc/world/classic/worldgen/WorldGenLooseRocks.java

```diff
@@ -29,23 +29,30 @@
 
 public class WorldGenLooseRocks implements IWorldGenerator
 {
-    @Override
-    public void generate(Random random, int chunkX, int chunkZ, World world, IChunkGenerator chunkGenerator, IChunkProvider chunkProvider)
+    /**
+     * Generate loose rocks in the specified chunk's X and Z coordinates.
+     * Attention: This function assumes this is on server and in a TFC world dimension.
+     * Make sure to only call this function if the above are true.
+     *
+     * @param random the Random obj
+     * @param chunkX the chunk X coordinate
+     * @param chunkZ the cunk Z coordinate
+     * @param world  the WorldObj
+     * @param factor the factor to use in this generation. 1.0 Means fully generate config looseRocksFrequency
+     * @param generateOres true if you want to generate small ores of veins found in this chunk
+     */
+    public static void generateLooseRocks(Random random, int chunkX, int chunkZ, World world, double factor, boolean generateOres)
```

Comment:
Maybe i should rewrite this properly and don't populate `List<Vein> veins` if generateOres isn't true.